No freaking way

Costpap opened the pull request:
New sound effect.

Costpap committed:
Added mp3 sound

Costpap committed:
Added No Freaking Way sound effect.

pep8speaks commented:
Hello @Costpap! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



There are currently no PEP 8 issues detected in this Pull Request. Cheers! :beers: 

##### Comment last updated at 2019-12-19 16:08:04 UTC

ks00908 commented on doot-doot/cogs/Airhorn.py:
```diff
@@ -228,6 +228,11 @@ def __init__(self, bot):
         """Honk Honk!"""
         await play_file(ctx, "sounds/honk.mp3")
 
+   @commands.command()
```
i think you missed space

ks00908 commented on doot-doot/cogs/Airhorn.py:
```diff
@@ -228,6 +228,11 @@ def __init__(self, bot):
+    @commands.guild_only()
+    async def nofreakingway(self, ctx):
+        """From the Megamind (2010) movie soundboard."""
+        await play_file(ctx, "sounds/no-freaking-way!.mp3")
```
for safety please remove ! from file, it technically shouldnt be an issue but i want to play it safe

Costpap replied:
The ! was in the mp3 name, so there shouldn't be an issue since it's the exact name of the file. I can reupload without the ! if needed, though.

Costpap committed:
Re-uploaded mp3 file without !

As requested in #24

Costpap committed:
I'm dumb and didn't upload this to the /sounds folder so deleting

Costpap committed:
Uploaded it to the right place this time

Costpap committed:
Delete no-freaking-way!.mp3

As requested in PR #24

Costpap committed:
Fixed spacing

As requested in PR #24

Costpap committed:
Removed ! from file name

Forgot to do it in commit 383eb4d48c721b7ae2794c5d4f6029091e95c94d so I had to make a new one

ks00908 committed:
Merge branch 'master' into no-freaking-way
